### DISCUSSION TITLE
Automatically close nemo when idle after X minutes or seconds (or hours)

### DISCUSSION THREAD

**Discussion by developer1105** [linuxmint/discussions#370] [Ideas]
Well, this feature request was inspired by this web page:
     
_"Automatically close terminal if not used for a while"_

I'm sending only the page title so you can check for yourselves.
The page is hosted at medium(dot)com
     
The idea is to implement some kind of security, because leaving nemo window open all the time may leave the filesystem available somehow to anyone who may have access to the computer at that particular moment. It may also help with some memory gain, although not that much.
     
Of course, this would be a feature turned off by default, and I guess it would be mainly aimed at power users.

**Comment by claudiux** [comment]
It seems to me that this is more the role of the screensaver.

**Comment by JosephMcc** [comment]
I don't see the point. If your computer is unlocked anyone can access it. What does closing Nemo accomplish? Someone can simply reopen it.
